src/main/java/com/itacademy/cms/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.itacademy.cms.mapper;
+
+import com.itacademy.cms.model.Category;
+import com.itacademy.cms.model.dto.CategoryDto;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(componentModel = "spring")
+public interface CategoryMapper {
```

Comment:
please use one "MapStructMapper" config interface for all mappings

Reply:
Created one "MapStructMapper" config interface for all mappings